### PR TITLE
fix text align of the arabic readme.md and rename fille to remane consistent and some synthax errors

### LINKED ISSUE #589: Translate README.md to Arabic Language
Hello my native language is Arabic, and I would like to translate the readme.md to the Arabic Language .

### PR DESCRIPTION
**Pull request by marzouk2002**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

Closes #589

### CODE REVIEW

**Comment by erictleung** [comment]
@marzouk2002 thanks for your contribution and translation. There appears to be some work on this PR that could use some review and was created a while ago. Please take a look at see how this translation is https://github.com/freeCodeCamp/how-to-contribute-to-open-source/pull/457

File: README-AR.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+</details>
+<!-- Do not translate this -->
+
+# اهلا بك في المساهمين المبتدئين في المصادر مفتوحة المصدر  
+
+[![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
+[![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
+[![Build Status](https://api.travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
+
+هذه مجمزهة من المصادر مفتوحة المصدر للشباب الذين يريدون المساهمة فالمشاربع مفتوحة المصدر
```
Comment (Green-Abe) [inline README-AR.md:32]: مجموعة

File: README-AR.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+# اهلا بك في المساهمين المبتدئين في المصادر مفتوحة المصدر  
+
+[![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
+[![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
+[![Build Status](https://api.travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
+
+هذه مجمزهة من المصادر مفتوحة المصدر للشباب الذين يريدون المساهمة فالمشاربع مفتوحة المصدر 
+
+اذا وجدت اي مصادر اخري مفيدة برجاء عمل طلب سحب لوضعها فالمصادر وشكرا لك
```
Comment (Green-Abe) [inline README-AR.md:34]: أخرى

**Comment by erictleung** [comment]
Closing in favor of https://github.com/freeCodeCamp/how-to-contribute-to-open-source/pull/670

**Pull request closed without merging**